apps/block_scout_web/test/block_scout_web/controllers/api/v2/address_controller_test.exs

```diff
@@ -1621,6 +1623,60 @@ defmodule BlockScoutWeb.API.V2.AddressControllerTest do
     end
   end
 
+  describe "/addresses/{address_hash}/forward-transfers" do
+    test "get empty list on non existing address", %{conn: conn} do
+      address = build(:address)
+
+      request = get(conn, "/api/v2/addresses/#{address.hash}/forward-transfers")
+
+      assert %{"message" => "Not found"} = json_response(request, 404)
+    end
+
+    test "get 422 on invalid address", %{conn: conn} do
+      request = get(conn, "/api/v2/addresses/0x/forward-transfers")
+
+      assert %{"message" => "Invalid parameter(s)"} = json_response(request, 422)
+    end
+
+    test "get forward transfer", %{conn: conn} do
+
+      forward_transfer = insert(:forward_transfer)
+
+      request = get(conn, "/api/v2/addresses/#{forward_transfer.to_address_hash}/fee-payments")
+      assert response = json_response(request, 200)
+      assert Enum.count(response["items"]) == 1
+      assert response["next_page_params"] == nil
+      compare_item(forward_transfer, Enum.at(response["items"], 0))
+    end
+  end
+
+  describe "/addresses/{address_hash}/fee-payments" do
+    test "get empty list on non existing address", %{conn: conn} do
+      address = build(:address)
+
+      request = get(conn, "/api/v2/addresses/#{address.hash}/fee-payments")
+
+      assert %{"message" => "Not found"} = json_response(request, 404)
+    end
+
+    test "get 422 on invalid address", %{conn: conn} do
+      request = get(conn, "/api/v2/addresses/0x/fee-payments")
+
+      assert %{"message" => "Invalid parameter(s)"} = json_response(request, 422)
+    end
+
+    test "get fee payment", %{conn: conn} do
+
+      fee_payment = insert(:fee_payment)
+
+      request = get(conn, "/api/v2/addresses/#{fee_payment.to_address_hash}/fee-payments")
+      assert response = json_response(request, 200)
+      assert Enum.count(response["items"]) == 1
+      assert response["next_page_params"] == nil
+      compare_item(fee_payment, Enum.at(response["items"], 0))
+    end
+  end
+
```

Comment:
It would be good to test pagination - both for blocks and address filtered queries.
A good test would also be pagination where there are multiple fee payments from the same block, some which appear on the first page and some on the next.

See an example test here: https://github.com/HorizenLabs/eon-explorer/blob/WEB3-260/apps/block_scout_web/test/block_scout_web/controllers/api/v2/block_controller_test.exs#L287-L310

Reply:
yes agree, I added a test with pagination checks for both `blocks` and `addresses` unit tests module.
I also added the unit test modules `forward_transfer_controller_test.exs` and `fee_payment_controller_test.exs` to test the `/forward-transfers` and `/fee-payments` endpoints.